File: ProyectoV2022/Probando/Entities/Medico.cs
using ServicioMedico.Data;

namespace ServicioMedico.Entities
{
    public class Medico : Usuario
    {
        [Required]
        [StringLength(15)]
        public string numColegio { get; set; }

        public virtual List<Cita> citas { get; set; }

        public virtual List<Paciente> pacientes { get; set; }   
    }
}

File: googleoauth2/src/model/medico.ts
import cita from './Cita';
import medicoDTO from './MedicoDTO';
import pacienteDTO from './PacienteDTO';
import usuario from './UsuarioDTO';

interface medico extends medicoDTO {

	pacientes: pacienteDTO[],
	citas: cita[],

}

export default medico;